src/components/menu-bar/menu-bar.css

```diff
@@ -241,8 +214,16 @@
     vertical-align: middle;
 }
 
+.collapsible-label {
+    margin: 0 .5rem 0 .25rem;
+}
+
+[dir="rtl"] .collapsible-label {
```

Comment:
I feel like there should be some sort of annotation to say:
"instead of `top right bottom left`, treat this one as `top end bottom start`"
:P

Reply:
Totally! I think [`margin-inline`](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline) is basically what I want, but it looks like the browser support doesn't go far enough back.